libs/services/src/view/view.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@nestjs/common';
+import { VmQueryService } from '@libs/common/vm-query/vm-query.service';
+import { CacheService } from '@multiversx/sdk-nestjs-cache';
+import { CacheInfo, CommonConfigService } from '@libs/common';
+import { VmQueryArguments } from '@libs/common/vm-query/dtos/vm-query.arguments';
+import { GovernanceConfig } from '@libs/entities/entities/governance.config';
+import { GovernanceProposal } from '@libs/entities/entities/governance.proposal';
+import { GovernanceDelegatedVoteInfo } from '@libs/entities/entities/governance.delegated.vote.info';
+import { GovernanceContractService } from '../contracts/governance/governance.contract.service';
+import { GovernanceVotingPower } from '@libs/entities/entities/governance.voting.power';
+
+@Injectable()
+export class ViewService {
+  constructor(
+    private readonly vmQueryService: VmQueryService,
+    private readonly cacheService: CacheService,
+    private readonly commonConfigService: CommonConfigService,
+    private readonly governanceContractService: GovernanceContractService,
+  ) { }
+
+  async getGovernanceConfig(): Promise<GovernanceConfig> {
+    return await this.cacheService.getOrSet(
+      CacheInfo.GovernanceConfig.key,
+      async () => await this.getGovernanceConfigRaw(),
+      CacheInfo.GovernanceConfig.ttl,
+    );
+  }
+
+  async getGovernanceConfigRaw(): Promise<GovernanceConfig> {
+    const vmQueryResponse = await this.vmQueryService.query(new VmQueryArguments({
```

Comment:
use the governanceContractService.viewConfig method()

Reply:
right. done